CODE_OF_CONDUCT.md

```diff
@@ -1,62 +0,0 @@
-<div id="theia-logo" align="left">
-    <br />
-    <img src="https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/EF_GRY-OR_svg.svg?sanitize=true" alt="Eclipse Logo" width="300"/>
-</div>
-
-# Community Code of Conduct
```

Comment:
@planger I deleted this file as we don't have a CoC for CDT.cloud yet.

Reply:
Imho we should add the CoC of Eclipse to all CDT.cloud repositories.
https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php
But this doesn't have to be done right now with this initial contribution.